examples/util/base.js

```diff
@@ -18,6 +18,7 @@ const preferMap = {
 
 const imageClassificationModels = [{
   modelName: 'MobileNet v1 (TFLite)',
+  format: 'TFLite',
```

Comment:
My only concern is "format" property here, the original model format was parsed from "modelFormatName", and there are duplicated work after adding new property.

I agree to have new "format" property, but we may rethink about the "modelFormatName" and usages in examples and benchmarks.

Reply:
@ibelem I have rebased and replaced `modelFormatName` with `modelId`. Also, I changed the URL query parameters so that neither the URI nor the UX depends on the `modelId`. Please take a look at the latest commit, thanks.